src/test_report.py

```diff
@@ -6,10 +6,13 @@
 # Author: Jeny Sadadia <jeny.sadadia@gmail.com>
 #
 # Copyright (C) 2022 Collabora Limited
-# Author: Jeny Sadadia <jeny.sadadia@collabora.com>
```

Comment:
Please leave the line with Jeny, see the git history.

Reply:
updated